python/python/glide/async_commands/core.py

```diff
@@ -1322,6 +1322,38 @@ async def zcount(
             ),
         )
 
+    async def zpopmin(
+        self, key: str, count: Optional[int] = None
+    ) -> Mapping[str, float]:
+        """
+        Removes and returns the members with the lowest scores from the sorted set stored at `key`.
+        If `count` is provided, up to `count` members with the lowest scores are removed and returned.
+        Otherwise, only one member with the lowest score is removed and returned.
+
+        See https://redis.io/commands/zpopmin for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            count (Optional[int]): If not specified or None, pops one member.
```

Comment:
missing the description of this arg. something like
Count: "Specifies the quantity of members to pop. If not specified, pops one member."